operations/power_management/Power_On_and_Boot_Compute_Nodes_and_User_Access_Nodes.md

```diff
@@ -101,63 +101,63 @@ This procedure boots all compute nodes and user access nodes \(UANs\) in the con
 
 1. (`ncn-m001#`) List detailed information about the available boot orchestration service \(BOS\) session template names.
 
-    Identify the BOS session template names (such as `"cos-2.0.x"`, `slurm`, or `uan-slurm`), and choose the appropriate compute and UAN node templates for the power on and boot.
+    Identify the BOS session template names (such as `"compute-23.7.0"` or `uan-23.7.0`), and choose the appropriate compute and UAN node templates for the power on and boot.
 
     ```bash
-    cray bos v1 sessiontemplate list --format toml
+    cray bos sessiontemplates list --format toml | grep name
```

Comment:
BOSv2 is the default with CSM 1.4.

Reply:
I know. I'm just saying that if we document a bunch of commands without the `v1` or `v2` in them, and then there's a `v3`, and `v3` changes the commands in some backwards-incompatible way, then the commands without the `vN` in them will break, whereas commands that do explicitly specify the version in them will continue to work.

This was a sticking point when the default was changed from BOS v1 to v2 because of commands like `cray bos sessiontemplate list` turning into `cray bos sessiontemplates list`.

Anyway, it's a minor complaint. Hopefully there is not another version change that breaks backwards compatibility in this way.

Reply:
Moving to json and jq.